packages/core/src/routes/interaction/index.ts

```diff
@@ -35,7 +39,22 @@ export default function interactionRoutes<T extends AnonymousRouter>(
   );
 
   router.post(
-    '/passcode/send',
+    `${verificationPrefix}/social/authorization_uri`,
```

Comment:
To align with our naming convension for route, suggest using `authorization-uri`

Reply:
updated